samples/recognize.v1p1beta1.js

```diff
@@ -245,6 +245,65 @@ function syncRecognizeWithMetaData(
   // [END speech_transcribe_file_with_metadata]
 }
 
+function syncRecognizeWithEnhancedModel(
+  filename,
+  encoding,
+  sampleRateHertz,
+  languageCode
+) {
+  // [START speech_transcribe_file_with_enhanced_model]
+  // Imports the Google Cloud client library
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library for Beta API
+  /**
+   * TODO(developer): Update client library import to use new
+   * version of API when desired features become available
+   */
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const filename = 'Local path to audio file, e.g. /path/to/audio.raw';
+  // const encoding = 'Encoding of the audio file, e.g. LINEAR16';
```

Comment:
in python, the encodings are available as an enum: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/speech/google/cloud/speech_v1p1beta1/gapic/enums.py#L18

I didn't see one for node. Is there somewhere we can direct users to possible encodings?

Reply:
Yes, I see them in the documents mentioned here:
https://cloud-dot-devsite.googleplex.com/nodejs/docs/reference/speech/1.4.x/google.cloud.speech.v1#.AudioEncoding

Do you think this suffices, or do we need to provide the link in the sample code? In my opinion, the doc is good enough, but I am not against having the link in the sample code either.

Reply:
I am satisfied with it as is if you are. I do tend to prefer constants/enums to magic strings in client libs since it avoids oddness (I expect a user to enter `encoding = 'mp3'` for instance) but this is probably out of scope of writing a sample :)

Reply:
Yeah, agreed. enums are much better in such cases.   :)
Okay, I guess we can leave it as is then.

Reply:
Yep, leave it as is, this is just how the Node GAPIC works at the moment _(issues/feature requests can be filed @ [googleapis/toolkit](https://github.com/googleapis/toolkit))_

---

**#fyi #seealso**

How to pass enum values with generated GAPIC libraries?
 - C# :: **native enums** are used, pass a C# enum
 - Ruby :: use a **symbol**, eg. :the_value (snakecase) or :THE_VALUE (original proto casing, uppercase)
 - JavaScript :: use a **string**, eg. "THE_VALUE"
 - Python :: **native enums** (in Py3 only? Py2 emulated with simple types?)
 - ...

Depends on idioms 🤷🏼‍♀️

> Okay, I guess we can leave it as is then.

SGTM :)